Build/ProjectTypes/Analyzer.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
Not sure what determines whether the analyzers need to be included as a project reference now. The build still worked before that, still made a commit where it's updated.

Reply:
I recommend wie ask Marcel about the reason. Maybe he still remembers? It's possible there was a benefit to having native .net versions instead of netstandard?
What happens if you keep only netstandard? do the tests still execute green and does the analyzer work?

Reply:
The analyzer works, tests are green, as we have green builds on TC.

Reply:
The SafeContext analyzers defined the .Net6 version in their .csproj first (as far as I can find...). As I wrote those, I think it just comes down to me not knowing 100% what was going on in .csproj files and what the different .net versions were at that time. It seems it was just a mistake.

Reply:
Nevermind, those safecontext analyzers were based on the plaintext/webstring analyzers, talking to Marcel rn.

Reply:
Marcel also does not remember why the .net native versions were also targeted, so we can probably just remove them.